apps/site/assets/ts/shuttles/__tests__/ShuttlesPageTest.tsx

```diff
@@ -73,3 +85,116 @@ describe("it passes to ShuttlesOverview", () => {
     });
   });
 });
+
+describe("the selected", () => {
```

Comment:
Just to put what we were talking about yesterday in writing: we should document (in a comment here perhaps) exactly why we're doing this odd arrangement of `describe` vs `it` so no one gets confused when they try to refactor it in the future.

Reply:
Will address in my next PR.